core/templates/services/literally-canvas-helper.service.spec.ts

```diff
@@ -274,4 +269,9 @@ describe('LiterallyCanvasHelperService', function() {
     );
     expect(svgTag).toBe(openSvgTag);
   });
+
+  it('should fail svg validation', function() {
+    var svgTag = '<svg xmlns="http://www.w3.org/2000/svg" width="450" height="350" viewBox="0 0 450 350"> <rect width="450" height="350" x="0" y="0" fill="transparent" /> <g transform="translate(0, 0)"> <rect id="rectangle-cd0a7974-e7da-d040-90b3-47bc4737a744" x="44" y="55.125" widht="72" height="55" stroke="hsla(0, 0%, 0%, 1)" fill="hsla(0, 0%, 100%, 1)" stroke-width="2"></rect> </g><script src="evil.com"></script> </svg>';
```

Comment:
Should widht be width?

Also, given this -- is it possible to get a better indication of why it failed? For all you know, it could have failed due to widht not being an allowed attr. I think the best thing to do here is to create a tag that is as simple as possible but is still valid, add an expectation that it is valid, then modify it slightly to make it bad, and add an expectation that the new form is bad. That way you can be confident that the badness is due to what changed between the two versions.

Reply:
I have added additional tests for both attribute and tag. Done.